book/docs/dyn_tutorial/labeled_arguments.md

```diff
@@ -4,50 +4,49 @@ sidebar_position: 3
 
 # Aside: Labeled arguments
 
-Before we go further with the tutorial, it's worth nothing that Dada supports *labeled arguments*. That means that instead of writing `Point(22, 44)` one can also give labels to each argument, like `Point(x: 22, y: 44)`:
+Before we go further with the tutorial, it's worth nothing that Dada supports _labeled arguments_. That means that instead of writing `Point(22, 44)` one can also give labels to each argument, like `Point(x: 22, y: 44)`:
```

Comment:
```suggestion
Before we go further with the tutorial, it's worth noting that Dada supports _labeled arguments_. That means that instead of writing `Point(22, 44)` one can also give labels to each argument, like `Point(x: 22, y: 44)`:
```

lol, that's a funny one